demos/supabase-todolist/README.md

```diff
@@ -21,7 +19,7 @@ A step-by-step guide on Supabase<>PowerSync integration is available [here](http
 2. Open the repo in Android Studio. This creates a `local.properties` file in root and should contain a `sdk.dir=/path/to/android/sdk` line.
```

Comment:
@DominicGBauer I have not been able to test this demo app following these instructions (but I think it may be isolated to my dev setup) - can you confirm they work for you?

Reply:
@benitav yes it works for me.